src/Middleware/WebSockets/src/WebSocketMiddlewareExtensions.cs

```diff
@@ -7,8 +7,20 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// IApplicationBuilder extensions methods to add and configure WebSocketMiddleware.
```

Comment:
Nit: missed this earlier.

```suggestion
    /// IApplicationBuilder extension methods to add and configure WebSocketMiddleware.
```

Reply:
Damn! I missed that too. Sorry for that!